docker/yarn/build_docker_image.bash

```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
 
-image_name="cluster/hadoop"
-image_name="$image_name:new"
+image_name="spiritlab/sparkfhe-hadoop-cluster"
+# image_name="$image_name:new"
```

Comment:
you might want to remove this line, by default it will be "latest"

Reply:
Would we possibly need it in future for version control of docker images?